README.md

```diff
@@ -61,3 +61,15 @@ $ npm install
 ```shell
 $ npm run build
 ```
+
+## How to install
```

Comment:
I think that the install command should be written before the build command, since most users only need the install command.

Reply:
Fixed readme.